lang/en.json

```diff
@@ -144,12 +144,14 @@
 "DND5E.AdvancementScaleValueTitle": "Scale Value",
 "DND5E.AdvancementScaleValueHint": "A single value that changes as a class increases in level that is made available in roll formulas (such as a Monk's martial arts die).",
 "DND5E.AdvancementScaleValueIdentifierHint": "This scale value for the current level will be available using <strong>@scale.{class}.{identifier}</strong> in a formula.",
+"DND5E.AdvancementScaleValueTypeCR": "Challenge Rating",
 "DND5E.AdvancementScaleValueTypeDice": "Dice",
 "DND5E.AdvancementScaleValueTypeDistance": "Distance",
-"DND5E.AdvancementScaleValueTypeHintString": "Use this if you are unsure of the type of the scale values, or if the type does not matter.",
-"DND5E.AdvancementScaleValueTypeHintNumber": "Use this for purely numerical values like rage damage bonus or number of uses per day.",
+"DND5E.AdvancementScaleValueTypeHintCR": "Use this for representing creature challenge ratings for features like Wildshape.",
```

Comment:
I don't know how consistent we've been with this in the past but the Druid feature is actually two words: 'Wild Shape'.